src/main/java/com/urlparser/Main.java

```diff
@@ -0,0 +1,15 @@
+package com.urlparser;
+
+import com.urlparser.entity.Links;
+
+public class Main {
+
+  public static void main(String[] args) {
```

Comment:
@undsett how about this usage:
```
Links links = new LinksFromURL(url);
for (URL url: links) {
    System.out.println(url);
}
```

Reply:
Why usage of cycle FOR is better then links.listed() call?